dockers/docker-database/docker-database-init.sh

```diff
@@ -59,7 +71,7 @@ if [[ $DATABASE_TYPE == "chassisdb" ]]; then
 fi
 
 # copy/generate the database_global.json file if this is global database service in multi asic platform.
-if [[ $NAMESPACE_ID == "" ]] && [[ $NAMESPACE_COUNT -gt 1 ]]
+if [[ $DATABASE_TYPE == "" ]] && [[ $NAMESPACE_COUNT -gt 1 || $NUM_DPU -gt 1 ]]
```

Comment:
In the DPU scenario, all dpu databases are running in the host namespace which means the namespace_id is also empty to the dpu database instance. So, I use database_type == "" to identify the host db instance.

Reply:
From what I understand in case of dpu we set the DATABASE_TYPE as below, and for all other cases like single_asic/multi-asic DATABASE_TYPE will be ""

The earlier check below was applicable only to global db which is in host namespace in case of multi-asic/single asic 
```
if [[ $NAMESPACE_ID == "" ]] && [[ $NAMESPACE_COUNT -gt 1 ]]
```

So now with this change ```$NAMESPACE_ID == ""```  -------->  ```$DATABASE_TYPE == ""``` , for multi-asic platforms it will never go to the else part as DATABASE_TYPE is always "" ?.

Reply:
You are right, I will fix it.

Reply:
Fix it in PR: https://github.com/sonic-net/sonic-buildimage/pull/17221 Please help to check it.

Reply:
Taking a look.